net/routing/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=       quagga
+PLUGIN_VERSION=    0.0.1
+PLUGIN_COMMENT=    Plugin to manage the quagga routing daemon software which offers routing protocols like RIP, IS-IS and OSPF to guide your packets through the sea
+PLUGIN_MAINTAINER= franz.fabian.94@gmail.com
+PLUGIN_DEVEL =     devel
```

Comment:
Actually not so bad, it allows us to merge the code to stable/17.1 without building it. It's better for review. For style the argument should be "yes". 👍